src/hooks/data/concentrators/useConcentratorVaults.ts

```diff
@@ -15,11 +16,11 @@ const filterCategoryByIndex: FilterCategory[][] = [
   ["balancer"],
   ["curve"],
 ]
-const targetAssetByIndex: ConcentratorTargetAsset[] = [
-  "auraBAL",
-  "cvxCRV",
-  "ETH",
-  "ETH",
+const targetAssetByIndex: Address[] = [
+  "0x616e8BfA43F920657B3497DBf40D6b1A02D4608d",
```

Comment:
I know this is temporary but could you use a variable to describe these addresses? It will make the code easier for someone else